vars/sconsBuild.groovy

```diff
@@ -156,19 +191,26 @@ def call(Map config = [:]) {
     if (config['skip_clean']) {
         clean_cmd += "echo 'skipping scons -c'\n"
     } else {
-        def clean_files = "_build.external{,-Linux}"
+        def clean_files = "_build.external"
         if (config['clean']) {
             clean_files = config['clean']
         }
         clean_files += ' install build {daos_m,daos,iof,cart-Linux}.conf'
         clean_files += ' .sconsign{,-Linux}.dblite .sconf-temp{,-Linux}'
         clean_cmd += scons_exe + " -c ${sconstruct}\n"
         clean_cmd += "rm -rf ${clean_files}\n"
-        if (config['target']) {
-            clean_cmd += "rm -rf ${config['target']}/${clean_files}\n"
+        if (config_target) {
+            clean_cmd += "rm -rf ${config_target}/${clean_files}\n"
         }
     }
 
+    // Builds of Raft may fail if this directory is present in the workspace
+    // from a previous run.
+    // Builds may fail if bandit.xml files present in the workspace.
+    sh label: 'Remove some old build files if present.',
+       script:'rm -rf src/rdb/raft/CLinkedListQueue bandit.xml test.cov',
+       returnStatus: true
+
```

Comment:
Aren't these just additional `clean_files`?

Reply:
They could be considered that.  But that other clean files step can be skipped and if these are not cleaned up builds can fail.   There is a ticket to fix the makefile for the raft file.